nautobot/extras/signals.py

```diff
@@ -21,7 +22,10 @@
 from .models import CustomField, DynamicGroup, DynamicGroupMembership, GitRepository, JobResult, ObjectChange
 from .webhooks import enqueue_webhooks
 
-logger = logging.getLogger(__name__)
+
+# thread safe change context state variable
+change_context_state = contextvars.ContextVar("change_context_state", default=None)
+logger = logging.getLogger("nautobot.extras.signals")
```

Comment:
Should probably revert this:

```suggestion
logger = logging.getLogger(__name__)
```